src/Microsoft.Diagnostics.Runtime.Tests/src/Helpers.cs

```diff
@@ -84,11 +84,11 @@ public static ClrAppDomain GetDomainByName(this ClrRuntime runtime, string domai
             return runtime.AppDomains.Single(ad => ad.Name == domainName);
         }
 
-        public static ClrModule GetModule(this ClrRuntime runtime, string filename)
+        public static ClrModule GetModule(this ClrRuntime runtime, string fileName)
```

Comment:
As a personal preference, I hate `fileName`, but I suppose I will live with it because it is more consistent.  =)